plutus-emulator/src/Wallet/Emulator/Wallet.hs

```diff
@@ -105,93 +101,88 @@ emptyWalletState :: Wallet -> WalletState
 emptyWalletState w = WalletState pk where
     pk = walletPrivKey w
 
-data WalletEffect r where
-    SubmitTxn :: Tx -> WalletEffect ()
-    OwnPubKey :: WalletEffect PubKey
-    UpdatePaymentWithChange :: Value -> (Set.Set TxIn, Maybe TxOut) -> WalletEffect (Set.Set TxIn, Maybe TxOut)
-    WalletSlot :: WalletEffect Slot
-    WalletLogMsg :: T.Text -> WalletEffect ()
-    OwnOutputs :: WalletEffect UtxoMap
-makeEffect ''WalletEffect
-
-type WalletEffs = '[CI.ChainIndexEffect, NC.NodeClientEffect, State WalletState, Error WAPI.WalletAPIError, Writer [WalletEvent]]
+type WalletEffs = '[NodeClientEffect, ChainIndexEffect, State WalletState, Error WAPI.WalletAPIError, Writer [WalletEvent]]
```

Comment:
Wait, didn't we define something like this already?

Reply:
There is a `WalletEffects`, which also has `WalletEffect` but doesn't have the `Error`, `Writer` and `State` effects. Sinec `WalletEffs` was only used in one place I just inlined it.